Compiler error from Stainless library on OSX 10.12.6

```Attempting to use Stainless for integration testing. Rust compiler complains:

Compiling stainless v0.1.11
error[E0063]: missing field `tokens` in initializer of `syntax::ast::Item`
   --> /Users/jimdandy/.cargo/registry/src/github.com-1ecc6299db9ec823/stainless-0.1.11/src/generate.rs:113:11
    |
113 |         P(ast::Item {
    |           ^^^^^^^^^ missing `tokens`

error[E0063]: missing field `tokens` in initializer of `syntax::ast::Item`
   --> /Users/jimdandy/.cargo/registry/src/github.com-1ecc6299db9ec823/stainless-0.1.11/src/generate.rs:149:11
    |
149 |         P(ast::Item {
    |           ^^^^^^^^^ missing `tokens`

error: aborting due to 2 previous errors

error: Could not compile `stainless`.
```

Met the same problem to compile stainless, both in windows & in MacOSX. Please fix it, thanks. 
```
error[E0063]: missing field `tokens` in initializer of `syntax::ast::Item`
   --> /Users/jchen/.cargo/registry/src/mirrors.ustc.edu.cn-15f9db60536bad60/stainless-0.1.11/src/generate.rs:113:11
    |
113 |         P(ast::Item {
    |           ^^^^^^^^^ missing `tokens`
error[E0063]: missing field `tokens` in initializer of `syntax::ast::Item`
   --> /Users/jchen/.cargo/registry/src/mirrors.ustc.edu.cn-15f9db60536bad60/stainless-0.1.11/src/generate.rs:149:11
    |
149 |         P(ast::Item {
    |           ^^^^^^^^^ missing `tokens`

   Compiling num-bigint v0.1.40
error: aborting due to 2 previous errors
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -17,6 +17,7 @@ authors = [
   "Valerii Hiora <valerii.hiora@gmail.com>",
   "Vladimir Pouzanov <farcaller@gmail.com>",
   "Emīls Piņķis <pinkisemils@gmail.com>",
+  "Roman Pearah <github.to.soma@neverbox.com>",
 ]
 description = "Organized, flexible testing framework."
 repository = "https://github.com/reem/stainless"

--- a/src/generate.rs
+++ b/src/generate.rs
@@ -133,7 +133,8 @@ impl<'a> Generate<&'a DescribeState> for Test {
             ),
             // Inherited visibility (not pub)
             vis: ast::Visibility::Inherited,
-            span: sp
+            span: sp,
+            tokens: None,
         })
     }
 }
@@ -176,7 +177,8 @@ impl Generate<()> for Bench {
             ),
             // Inherited visibility (not pub)
             vis: ast::Visibility::Inherited,
-            span: sp
+            span: sp,
+            tokens: None,
         })
     }
 }

